{"uuid":"3c0be4e1-37e7-4763-a3ae-0750964a356c","name":"Scenario Outline: The user getting error message with invalid python code for Types of Linked List","children":["6a3284f2-fdf9-4bde-b5db-5a093f55e589"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"9a13dbb4-8886-44b9-afca-5c7644adfa80-attachment","type":"image/png"},{"name":"Myscreenshot","source":"223043d0-f17c-4f1c-9ea2-877b64fdebdd-attachment"}],"parameters":[],"start":1696887723960,"stop":1696887724147},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696887724147,"stop":1696887724148},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696887724148,"stop":1696887724148},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696887724149,"stop":1696887724149}],"start":1696887723957,"stop":1696887724149}